cmd/admin-monitor.go

```diff
@@ -0,0 +1,202 @@
+/*
+ * Minio Client (C) 2019 Minio, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package cmd
+
+import (
+	"fmt"
+	"net/url"
+
+	humanize "github.com/dustin/go-humanize"
+	"github.com/fatih/color"
+	"github.com/minio/cli"
+	json "github.com/minio/mc/pkg/colorjson"
+	"github.com/minio/mc/pkg/console"
+	"github.com/minio/mc/pkg/probe"
+	"github.com/minio/minio/pkg/madmin"
+)
+
+var (
+	adminMonitorFlags = []cli.Flag{}
+)
+
+var adminMonitorCmd = cli.Command{
+	Name:            "monitor",
+	Usage:           "monitor cpu, mem, net and disk statistics",
+	Action:          mainAdminMonitor,
+	Before:          setGlobalsFromContext,
+	Flags:           append(adminMonitorFlags, globalFlags...),
+	HideHelpCommand: true,
+	CustomHelpTemplate: `Name:
+  {{.HelpName}} - {{.Usage}}
+
+USAGE:
+  {{.HelpName}} TARGET
+
+FLAGS:
+  {{range .VisibleFlags}}{{.}}
+  {{end}}
+EXAMPLES:
+  1. Get server cpu, mem, net and disk statistics of the 'play' server.
+       $ {{.HelpName}} play/
```

Comment:
1. I am able to run `mc admin monitor play extraArg`. This should exit with invalid argument error.
```
➜  mc git:(pr2661) ✗ mc admin monitor play extraArg
●  play.minio.io:9000    

   CPU        min        avg      max
   current    14.63%      14.63%    14.64%
   historic   0.03%      2.25%    462.47%

   MEM        usage
   current    467MiB
   historic   439MiB

```

2.Alignment is off for `avg` and `max` columns.

Reply:
Good catch on #1. Will fix both issues